### PR TITLE
Minor Ui bug fixes (merge 1st)

### LINKED ISSUE #417: Clickable rows in a table should have cursor: pointer

### PR DESCRIPTION
**Pull request by peterszerzo**
Fixes #417, #414 and the duplicate breadcrumbs slashes bug in the image below:

screen shot 2018-04-30 at 11 37 57 am

### CODE REVIEW

File: packages/website/src/Sections/Components/Breadcrumbs.tsx [hunk 1]
```diff
@@ -7,8 +7,14 @@ export const docsUrl = "https://github.com/contiamo/operational-ui/blob/master/d
 
 export const Component = () => (
   <Breadcrumbs>
-    <Breadcrumb>Link one</Breadcrumb>
-    <Breadcrumb>Link two</Breadcrumb>
+    {null}
```
Comment (TejasQ) [inline packages/website/src/Sections/Components/Breadcrumbs.tsx]: I'm reading this going.... _what?_ 🤔

File: packages/components/src/Breadcrumbs.tsx [hunk 2]
```diff
@@ -28,12 +28,15 @@ const Breadcrumbs = (props: Props) => (
       /* This IIFE adds the divider elements containing slashes between children, e.g:
        * <Breadcrumb>1</Breadcrumb> <Breadcrumb>2</Breadcrumb> -> <span>1</span> <span>/</span> <span>2</span> 
        */
-      const newChildren: React.ReactNode[] = []
+      const newChildren: React.ReactNode[] = [<Divider key={"breadcrumbdivider-leading"}>/</Divider>]
       const childrenCount = React.Children.count(props.children)
       React.Children.forEach(props.children, (child: React.ReactNode, index: number) => {
+        if (child === null || child === undefined || child === false) {
```
Comment (TejasQ) [inline packages/components/src/Breadcrumbs.tsx]: ? Can a boolean coercion simplify this statement?

File: packages/components/src/Breadcrumbs.tsx [hunk 3]
```diff
@@ -28,12 +28,15 @@ const Breadcrumbs = (props: Props) => (
...
+          return
+        }
         newChildren.push(child)
         if (index < childrenCount - 1) {
-          newChildren.push(<Divider key={"breadcrumbdivider-" + index}>/</Divider>)
+          newChildren.push(<Divider key={`breadcrumbdivider-${index}`}>/</Divider>)
```
Comment (TejasQ) [inline packages/components/src/Breadcrumbs.tsx]: Can we please avoid mutation?

File: packages/website/src/Sections/Components/Breadcrumbs.tsx (hunk 1 above)
Comment (peterszerzo) [reply packages/website/src/Sections/Components/Breadcrumbs.tsx]: It was test code to see if unnecessary slashes get inserted if I add null children. I agree it shouldn't be in website code though :)

File: packages/components/src/Breadcrumbs.tsx (hunk 3 above)
Comment (peterszerzo) [reply packages/components/src/Breadcrumbs.tsx]: I'll take the dare :)

**Pull request merged**